azkaban-common/src/test/java/azkaban/project/JdbcProjectImplTest.java

```diff
@@ -103,6 +103,21 @@ public void testCreateProject() throws Exception {
     Assert.assertEquals(project.getLastModifiedUser(), "testUser1");
   }
 
+  @Test
+  public void testCreateProjectsWithDifferentCases() {
+    final String projectName = "mytestproject";
+    final String projectDescription = "This is my new project with lower cases.";
+    final User user = new User("testUser1");
```

Comment:
Is flowname also in-sensive today in mysql database?

Reply:
Yes. Mysql database is case insensitive by default.

Reply:
Thanks for adding new test cases